lib/core/test/unit/Cardano/Wallet/Primitive/AddressDerivationSpec.hs

```diff
@@ -337,7 +339,20 @@ instance {-# OVERLAPS #-} Arbitrary (Passphrase "encryption") where
     arbitrary = do
         let p = Proxy :: Proxy "encryption"
         n <- choose (passphraseMinLength p, passphraseMaxLength p)
-        bytes <- T.encodeUtf8 . T.pack <$> replicateM n arbitraryPrintableChar
+        bytes <-
```

Comment:
this one is added as `n` can be valid, but lenght of bytes can exceed the upper bound after packing and encoding. Real examples,
```
n:194
bytes length:217

n:64
bytes length:75
```

Reply:
But, it doesn't matter for the encryption passphrase, does it?

Reply:
frankly speaking this line in `master` made me also touch encryption passphrase : https://github.com/input-output-hk/cardano-wallet/blob/master/lib/core/src/Cardano/Wallet/Primitive/AddressDerivation.hs#L246
it suggests that it should be minimum 10 characters

I will revert my changes concerning encryption passprhrase. Just focus on seed passphrase